mon-pix/app/components/qrocm-proposal.js

```diff
@@ -31,10 +31,4 @@ export default class QrocmProposal extends Component {
   onInputChange() {
     this.args.answerChanged();
   }
```

Comment:
```suggestion
  }

  @action
  onChange(key, value) {
    this.args.answersValue[key] = value;
    this.args.onChangeSelect(this.args.answersValue);
  }
```

On doit pouvoir éviter de muter `answersValue` directement vu que c'est le composant parent qui s'en charge mais je ne sais pas comment cloner cet objet.